### PR TITLE
Verbose out

### LINKED ISSUE #5: Verbose output
### Description
We may not want to always print debug information. For testing, it is useful. It would be nice to have perhaps 2 levels of verbosity:
* `default` (printed nicely)
* `-v`: increased verbosity for debugging (may not be printed nicely)

### Notes
Requires #6

### PR DESCRIPTION
**Pull request by BobbyBytes**
## Implements the (-v , --verbose) flag. 
* Closes #5 

Suppress the Client / Server reading / writing bytes messages 
I decided that currently only the messages where the client / server were displaying Bytes written/read were "verbose"

# Implementation

Implements Logging library for debug messages. 
By Default library prints ERROR or CRITICAL Level 
with verbose option it will be set to DEBUG level displaying all messages.

### CODE REVIEW

File: bin/securedrop_server [hunk 1]
```diff
@@ -15,4 +16,19 @@ if __name__ == "__main__":
...
-    server.main(filename=securedrop_file)
+    securedrop_port = None
+    try:
+        opts, args = getopt.getopt(sys.argv[1:], "pfv", ["port", "filename", "verbose"])
+    except getopt.GetoptError as err:
+        print(err)  # will print something like "option -a not recognized"
+        sys.exit(2)
+    for o, a in opts:
+        if o in ("-v", "--verbose"):
+            VerbosePrinter(True)
```
Comment (jjjoycing123) [inline bin/securedrop_server]: In the bin/securedrop file "VerbosePrinter(true)" is checked last in the if else chain. Here it is first. I think the logic here should match the bin/securedrop file.

**Comment by jjjoycing123** [comment]
Just had the one comment. All the rest looks fine to me!

**Comment by BobbyBytes** [comment]
Updated to use Logging Library :)

**Comment by BobbyBytes** [comment]
> Just had the one comment. All the rest looks fine to me!

OK, I put them in the same order.. I used the order of the arguments above.

**Review by aokellermann — CHANGES_REQUESTED** [review]
All print statements (besides that which is printed in client shell for user to read) should be using logger, using various levels. You only changed a few print statements here

**Comment by BobbyBytes** [comment]
Crap, I just realized the default level of Logging is WARNING so I need to change the default to INFO if we still want server to print what are now INFO level logs.

**Comment by BobbyBytes** [comment]
OK, I think this is good now.

**Review by aokellermann — APPROVED** [review]

**Pull request merged**